zjit/src/options.rs

```diff
@@ -67,6 +76,8 @@ pub const ZJIT_OPTIONS: &'static [(&str, &str)] = &[
     ("--zjit-num-profiles=num",   "Number of profiled calls before JIT (default: 1, max: 255)."),
     ("--zjit-stats",              "Enable collecting ZJIT statistics."),
     ("--zjit-perf",               "Dump ISEQ symbols into /tmp/perf-{}.map for Linux perf."),
+    ("--zjit-log-compiled-iseqs=path",
+     "Log compiled ISEQs to the specified file. The file will be truncated on each run."),
```

Comment:
This description is too long that it doesn't pass `TestRubyOptions#test_usage_long` in `test/ruby/test_rubyoptions.rb`.

You might want to align the indentation of this line to make the `80-char limit --> |` comment actually useful.